File: BookShop.BusinessLogic/Models/SearchParams.cs
using static BookShop.DataAccess.Entities.Enums.Enums.EntityFields;

namespace BookShop.BusinessLogic.Models
{
    public class SearchParams
    {
        public PrintedEditionType PrintedEditionType { get; set; }
        public SortCriteria SortCriteria { get; set; }
        public decimal PriceFrom { get; set; }
        public decimal PriceTo { get; set; }
        public int Page { get; set; }
        public int PageSize { get; set; }

    }
}

File: BookShop.ClientApp/src/app/printed-edition/models/searchParams.ts
import { SortCriteria } from '../enums/sortCriteria';

export class SearchParams {
    public priceFrom: number;
    public priceTo: number;
    public sortCriteria: SortCriteria;
    public page:number;
    public pageSize:number;

    constructor(
        ) { 
            this.priceFrom=0;
            this.priceTo=0;
            this.sortCriteria=SortCriteria.None;
        }
        
}